### PR TITLE
Explain tidb_max_tiflash_threads default value -1 means

### PR DESCRIPTION
**Pull request by llkdd1**
### What is changed, added or deleted? (Required)

translate docs to cn pr#15168

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/15168
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -2988,7 +2988,7 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 默认值：`-1`
 - 范围：`[-1, 256]`
 - 单位：线程
-- TiFlash 中 request 执行的最大并发度。默认值为 `-1`，表示该系统变量无效。`0` 表示由 TiFlash 系统自动设置该值。
+- TiFlash 中 request 执行的最大并发度。默认值为 `-1`，表示该系统变量无效,最大并发度取决于 TiFlash 配置文件 (`profiles.default.max_threads`) 的设置。`0` 表示由 TiFlash 系统自动设置该值。
```
Comment (shawn0915) [inline system-variables.md:2991]: >无效,最大
这里有个英文标点

Comment (llkdd1) [reply system-variables.md:2991]: 感谢

File: system-variables.md [hunk 2]
```diff
@@ -2988,7 +2988,7 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 默认值：`-1`
 - 范围：`[-1, 256]`
 - 单位：线程
-- TiFlash 中 request 执行的最大并发度。默认值为 `-1`，表示该系统变量无效。`0` 表示由 TiFlash 系统自动设置该值。
+- TiFlash 中 request 执行的最大并发度。默认值为 `-1`，表示该系统变量无效，最大并发度取决于 TiFlash 配置文件 (`profiles.default.max_threads`) 的设置。`0` 表示由 TiFlash 系统自动设置该值。
```
Comment (Oreoxmt) [inline system-variables.md:2991]:
```suggestion
- TiFlash 中 request 执行的最大并发度。默认值为 `-1`，表示该系统变量无效。最大并发度取决于 TiFlash 配置项 `profiles.default.max_threads` 的设置。`0` 表示由 TiFlash 系统自动设置该值。
```

**Review by Oreoxmt — APPROVED** [review]

File: system-variables.md [hunk 3]
```diff
@@ -2988,7 +2988,7 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 默认值：`-1`
 - 范围：`[-1, 256]`
 - 单位：线程
-- TiFlash 中 request 执行的最大并发度。默认值为 `-1`，表示该系统变量无效。`0` 表示由 TiFlash 系统自动设置该值。
+- TiFlash 中 request 执行的最大并发度。默认值为 `-1`，表示该系统变量无效。最大并发度取决于 TiFlash 配置项 `profiles.default.max_threads` 的设置。`0` 表示由 TiFlash 系统自动设置该值。
```
Comment (hfxsd) [inline system-variables.md:2991]: `profiles.default.max_threads` 是不是改成 `tidb_max_tiflash_threads` 比较好？参见 6.1.0 release notes https://docs.pingcap.com/zh/tidb/stable/release-6.1.0#%E6%98%93%E7%94%A8%E6%80%A7

TiFlash 配置项 max_threads 转化为系统变量 tidb_max_tiflash_threads ，从而支持配置项在线变更和持久化。转化后，原有配置项不会废弃。
```suggestion
- TiFlash 中 request 执行的最大并发度。默认值为 `-1`，表示该系统变量无效。最大并发度取决于 TiFlash 配置项 [`tidb_max_tiflash_threads`](/system-variables.md#tidb_max_tiflash_threads-从-v610-版本开始引入) 的设置。`0` 表示由 TiFlash 系统自动设置该值。
```

Comment (Oreoxmt) [inline system-variables.md:2991]:
```suggestion
- TiFlash 中 request 执行的最大并发度。默认值为 `-1`，表示该系统变量无效，此时最大并发度取决于 TiFlash 配置项 `profiles.default.max_threads` 的设置。`0` 表示由 TiFlash 系统自动设置该值。
```

**Review by hfxsd — APPROVED** [review]

**Pull request merged**